easybuild/easyconfigs/c/CSBDeep/CSBDeep-0.4.1-fosscuda-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,43 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
+
+name = 'CSBDeep'
+version = '0.4.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://csbdeep.bioimagecomputing.com/"
+description = """CSBDeep is a toolbox for Content-aware Image Restoration (CARE)."""
+
+toolchain = {'name': 'fosscuda', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('TensorFlow', '1.13.1', versionsuffix),
+    ('Keras', '2.2.4', versionsuffix),
+    ('IPython', '7.7.0', versionsuffix),
+    ('matplotlib', '3.0.3', versionsuffix),
+    ('tqdm', '4.32.1'),
+]
+
+use_pip = True
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    ('imagecodecs-lite', '2019.12.3', {
+        'checksums': ['95d18aa13ceb1b18a6109433b42d054e13b9a295cba96c08ab719f864f589d68'],
+    }),
+    ('tifffile', '2019.7.26.2', {
+        'checksums': ['2abb91c3a23a61593c5635ac1a19f67e732b46291c305fcee0eeaad41181a13f'],
+    }),
+    (name, version, {
+        'modulename': '%(namelower)s',
+        'source_tmpl': '%(namelower)s-%(version)s.tar.gz',
+        'checksums': ['ad2cf81389b8c15eb2f38a524dc011a5331dfb91e7d3053fed0c6c808a7bdf89'],
+    }),
+]
+
```

Comment:
@branfosj can you add the `sanity_check_commands = ['care_predict']` that was added to the foss version here too?

Reply:
done